### ISSUE TITLE
I would have preferred a fork...

### ISSUE THREAD

**Issue by aaronpropst** [justinhunt/moodlecst#1]
Hi Justin,   I was recently pointed to your code base here, and while I appreciate the attribution you put on the readme.md.. I would have preferred for you to fork the project proper.  As it is there is no link between the two, no reference that your first commit contains my code, and no ability for you to push any advances you've made back to the original CST project.

I would have gladly made you an owner/collaborator on the actual CST repo.  I don't have a lot of spare time for the project, and was hoping for someone to inherit it.

Handy reading on why forking is good for the social coding ethos of github:
https://guides.github.com/activities/forking/

Thanks!
Aaron

**Comment by justinhunt** [comment]
Yes, you are right. It should have been a fork. I wish I had done that now, and I apologize. In all honesty, it just never occurred to me at the time. I downloaded it to play with it a bit and just started developing on it. At some point I pushed it to github. I am a fairly basic git user and that is partly the explanation I guess.
